v23.1/cursors.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Cursors
+summary: Paginate results from queries against your cluster using SQL cursors
+toc: true
+---
+
+A cursor is a placeholder into a selection query that allows you to iterate over subsets of the rows returned by that query.
+
+Cursors differ from [keyset pagination](pagination.html) and [`LIMIT`/`OFFSET`](limit-offset.html) in that:
+
+- Each cursor is a stateful SQL object that is referred to by a unique name.
+- Each cursor requires holding open its own dedicated (read-only) [transaction](transactions.html).
+- Each cursor operates on a snapshot of the database at the moment that cursor is opened.
+
+## Synopsis
+
+Cursors are declared using the following keywords:
```

Comment:
Isn't just DECLARE used to declare a cusor? FETCH and CLOSE are used to interact with it not declare it.

Reply:
we can change to "Cursors are declared and used with the following keywords" or something

Reply:
Update to the language suggested by Rafi!